test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPackTests.cs

```diff
@@ -73,6 +80,8 @@ public void P2PDifferentFrameworks(DnxSdk sdk)
 
             // Assert
             Assert.Equal(0, result.ExitCode);
+
+            TestUtils.CleanUpTestDir<DnuPackTests>(sdk);
```

Comment:
Can't use the dispose method since I need to name of the test method in the clean up. This is run at the end of each test to delete any files copied from misc during the test. The coming up test can clean it but this cleans it eagerly.